README.md

```diff
@@ -1899,13 +1899,124 @@ and verify your domain. Generate an API key for the Mailgun account. Note your M
 
 #### Configure Mailgun
 
-Configure the Mailgun credentials in your .env file. Add your Mailgun key to the application.yaml as
-seen below:
+To configure Mailgun with the proper credentials an engineer must pass a mailgun domain,
+mailgun key, and a mailgun sender-email into their application. Those keys are passed into
```

Comment:
Maybe link to documentation telling folks where they can get their mailgun key?

https://help.mailgun.com/hc/en-us/articles/203380100-Where-Can-I-Find-My-API-Key-and-SMTP-Credentials-

Reply:
done